sdk/tests/deqp/framework/delibs/debase/deMath.js

```diff
@@ -48,6 +48,24 @@ deMath.deInBounds32 = function(a, mn, mx) {
 deMath.deFloatFrac = function(a) { return a - Math.floor(a); };
 
 /**
+ * Global variable used in function deMath.toFloat32()
+ */
+deMath.FLOAT32ARRAY1 = new Float32Array(1);
```

Comment:
Instead of defining this global variable, you can do it in a less polluting way.

deMath.toFloat32 =  (function(a) {
   var localArray = new Float32Array(1);
   return function(a) { localArray[0] = a; return localArray[0]; }; 
})();

(I might not be 100% semantically correct, but something like that)